### PR TITLE
Fix System.IO.Ports configurations and netfx build

### PR DESCRIPTION
**Pull request by ViktorHofer**
The netstandard-Windows_NT configuration always wins over netfx. Adding the RID to ensure that the configuration is building. Also moving an Interop file into the right ItemGroup which caused build erros.

### CODE REVIEW

File: src/System.IO.Ports/ref/Configurations.props [hunk 1]
```diff
@@ -2,8 +2,8 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
-      netfx;
-      net461;
+      netfx-Windows_NT;
```
Comment (ericstj) [inline src/System.IO.Ports/ref/Configurations.props]: Reference assemblies cannot have runtimes.

Comment (ericstj) [reply src/System.IO.Ports/ref/Configurations.props]: Maybe we should add an error for this case into some common targets file.

Comment (ViktorHofer) [reply src/System.IO.Ports/ref/Configurations.props]: done.

Comment (ericstj) [reply src/System.IO.Ports/ref/Configurations.props]: Central error so others don't make the same mistake?

File: src/System.IO.Ports/src/Configurations.props [hunk 2]
```diff
@@ -5,12 +5,12 @@
       netstandard-Linux;
       netstandard-OSX;
       netstandard;
-      net461;
+      net461-Windows_NT;
```
Comment (ericstj) [inline src/System.IO.Ports/src/Configurations.props]: Double check that the package still gets a RID-less asset.

File: src/System.IO.Ports/ref/Configurations.props (hunk 1 above)
Comment (ViktorHofer) [reply src/System.IO.Ports/ref/Configurations.props]: https://github.com/dotnet/corefx/issues/36568

**Comment by ViktorHofer** [comment]
I'm also going to fix the dangling ref for net472: https://github.com/dotnet/corefx/issues/36577 as part of this PR.

**Comment by ViktorHofer** [comment]
@ericstj sorry for asking for another review 😁. Please check https://github.com/dotnet/corefx/pull/36565/commits/85d9f58f8feb9bf3e7804f17f0a540f03721b5b8

File: src/System.IO.Ports/src/Configurations.props (hunk 2 above)
Comment (ViktorHofer) [reply src/System.IO.Ports/src/Configurations.props]: jep works as expected.

image

**Comment by ViktorHofer** [comment]
Build fails with netstandard2.0 instead of netstandard. Not sure why as I thought netstandard is a direct mapping to netstandard2.0 but I guess some conditions are wrong in externals?

I just changed it to netstandard.

**Comment by ericstj** [comment]
> Build fails with netstandard2.0 instead of netstandard. Not sure why as I thought netstandard is a direct mapping to netstandard2.0.

It still uses different paths since the paths are based on the literal string.  EG: bin\ref\netstandard and bin\ref\netstandard2.0.

**Pull request merged**